DXRModules/DeusEx/Classes/DXREventsBase.uc

```diff
@@ -1367,9 +1368,15 @@ function _MarkBingoAsFailed(coerce string eventname)
     } */
 
     data = class'PlayerDataItem'.static.GiveItem(player());
-    if (data.MarkBingoAsFailed(eventname) && class'MenuChoice_ShowBingoUpdates'.static.IsEnabled(dxr.flags)) {
+
+    if (data.MarkBingoAsFailed(eventname)) {
         l(self$"._MarkBingoAsFailed("$eventname$") data: "$data);
-        player().ClientMessage("Failed bingo goal: " $ data.GetBingoDescription(eventname));
+        if (class'MenuChoice_ShowBingoUpdates'.static.MessagesEnabled(dxr.flags) && dxr.localURL != "DX" && dxr.localURL != "DXONLY") {
+            player().ClientMessage("Failed bingo goal: " $ data.GetBingoDescription(eventname));
+        }
+        if (class'MenuChoice_ShowBingoUpdates'.static.SoundsEnabled(dxr.flags) && dxr.localURL != "DX" && dxr.localURL != "DXONLY") {
+            player().PlaySound(Sound'DeusExSounds.Generic.Buzz1', SLOT_None, 3.0);
```

Comment:
this is gonna be really annoying when you travel to the next mission and get 10 buzzes at the same time

keep a variable for last_buzz_time and set it to Level.TimeSeconds when playing the sound, don't play the sound if the last time is within like 0.1 of the current time

Reply:
you'll also want to set the variable to like -1 in defaultproperties, so it can buzz at the start of the level

Reply:
> this is gonna be really annoying when you travel to the next mission and get 10 buzzes at the same time

Yeah, I was wondering how to do something like that.